sdk/projects/azure-projects/setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from setuptools import setup, find_packages
+import os
+from io import open
+import re
+
+
+PACKAGE_NAME = "azure-projects"
+PACKAGE_PPRINT_NAME = "Projects"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace("-", "/")
+# a-b-c => a.b.c
+namespace_name = PACKAGE_NAME.replace("-", ".")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, "_version.py"), "r") as fd:
+    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE).group(1)
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+with open("README.md", encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description="Microsoft Azure {} Library for Python".format(PACKAGE_PPRINT_NAME),
+    # ensure that these are updated to reflect the package owners' information
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Azure/azure-sdk-for-python",
+    keywords="azure, azure sdk, azure projects",  # update with search keywords relevant to the azure service / product
+    author="Microsoft Corporation",
+    author_email="azuresdkengsysadmins@microsoft.com",
```

Comment:
Feels uneexpected

Reply:
lol - well spotted - that's left over from the template project.

Reply:
Updated.